Example/Podfile

```diff
@@ -1,9 +1,9 @@
 target 'PINRemoteImage', :exclusive => true do
-  pod "PINRemoteImage", :path => "../"
+  pod "PINRemoteImage/FLAnimatedImage", :path => "../"
```

Comment:
shouldn't this just be FLAnimatedImage?
Also, we shouldn't remove pod "PINRemoteImage", :path => "../", right?